holidays/countries/singapore.py

```diff
@@ -264,8 +269,10 @@ class SingaporeIslamicHolidays(_CustomIslamicHolidays):
         2022: (JUL, 10),
         2023: (JUN, 29),
         2024: (JUN, 17),
+        2025:(JUN, 7,),
```

Comment:
This is going to be flagged by pre-commit
```suggestion
        2025: (JUN, 7),
```